coverage.py

```diff
@@ -370,7 +374,7 @@ def process_one_provider(self, identifier, provider):
                 identifier, force=self.force
             )
         else:
-            coverage_record = provider.register(
+            coverage_record, is_new = provider.register(
```

Comment:
This was an error hidden by the fact that we don't test this with real CoverageProviders.